src/colorizer/ColorRamp.ts

```diff
@@ -1,10 +1,24 @@
-import { Color, ColorRepresentation, DataTexture, FloatType, LinearFilter, NearestFilter, RGBAFormat } from "three";
+import {
+  Color,
+  ColorManagement,
+  ColorRepresentation,
+  DataTexture,
+  FloatType,
+  LinearFilter,
+  NearestFilter,
+  RGBAFormat,
+} from "three";
 
 export enum ColorRampType {
   LINEAR,
   HARD_STOP,
 }
 
+// Fixes an issue in Three.js where the new default color management causes all
+// Color objects to be interpreted as sRGB colors, which breaks expected linear
+// interpolation between colors in the ramp.
+ColorManagement.enabled = false;
```

Comment:
ouch, this one is scary to me since afaik we don't do it in other apps. do you know what version of three introduced this?

Reply:
conversion to linear space is actually what we want, so that linear lookups of color textures do a more correct thing. (`texture.colorSpace`)

Reply:
Removed this and ended up just changing the unit test. 

I didn't explicitly change every `new Color()` or `Color.getHexString()` call to explicitly define the color space, since `three` assumes hex strings to be in sRGB when creating new colors and `Color.getHexString` defaults to sRGB. AFAIK it'll do the right thing, but if you think it's wise to be explicit LMK and I can make that change.